enh(mc): updated windows telegraf mc procedure (#3398)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/operatingsystems-windows-telegraf-agent.md

## Prerequisites

### Network flow

Two TCP flows must be open from the host to the poller.

| Source | Destination    | Protocole | Port | Purpose |
| ------ | -------------- | --------- | ---- | --- |

> | Hôte supervisé | Collecteur  | TCP       | 1443 | Access to Telegraf agent's configuration. |

Reviewer: ```suggestion
| Hôte supervisé | Collecteur  | TCP       | 1443 | Access to the configuration of the Telegraf agent. |
```

| Hôte supervisé | Collecteur  | TCP       | 4317 | OpenTelemetry data flow. |

### System prerequisites on the poller

> To be able to use the Telegraf agent, you must use a poller with at least version `24.04.2` of `centreon-engine`. The Telegraf agent will configure itself via a HTTPS request sent to Centreon Engine.

1. For this to work, you must first get a valid certificate or generate a self-signed one on the poller as detailed below.

> > In the next prerequisites, replace `${HOSTNAME}` with the poller's FQDN if they don't match.

Reviewer: ```suggestion
> In the command below, replace `${HOSTNAME}` with the poller's FQDN if they don't match.
```

```bash
openssl req -new -subj "/CN=${HOSTNAME}" -addext "subjectAltName = DNS:${HOSTNAME}" -newkey rsa:2048 -sha256 -days 365 -nodes -x509 -keyout /etc/centreon-engine/conf-server.key -out /etc/centreon-engine/conf-server.crt
chown centreon-engine: /etc/centreon-engine/conf-*
```

> > The `-days 365` option limits the certificate validity to one year. You may choose a longer or shorter duration according to your security/maintainance preferences.

Reviewer: ```suggestion
> The `-days 365` option limits the certificate's validity to one year. You may choose a longer or shorter duration according to your security/maintainance preferences.
```

> 2. Then define Engine's configuration server for Telegraf.

Reviewer: ?

Author: il s'agit d'un serveur (http) qui fournit la configuration à Telegraf

Reviewer: Define which server will provide the configuration to the agent.?

Author: Non, ce n'est pas la question, il s'agit de configurer le configurateur :-D

> 2. Then define Engine's configuration server for Telegraf.

Reviewer: ```suggestion
2. Then provide Engine with the connection information it needs to give to the Telegraf agent so that it can send information to Engine.
```

            "certificate_path": "/etc/centreon-engine/conf-server.crt",
            "key_path": "/etc/centreon-engine/conf-server.key"
        },
        "engine_otel_endpoint": "${HOSTNAME}:4317",
        "check_interval":60
    }
}
EOF
chown centreon-engine: /etc/centreon-engine/otl_server.json
```

### Configuration of Centreon Engine

> 1. In the **Configuration > Pollers > Engine configuration** menu, under the **Data** tab, add an entry to the Broker modules to load and enter the `/usr/lib64/centreon-engine/libopentelemetry.so /etc/centreon-engine/otl_server.json` directive. Save the form.

Reviewer: ```suggestion
1. In the **Configuration > Pollers > Engine configuration** menu, on the **Data** tab, add an entry to the Broker modules to load and enter the `/usr/lib64/centreon-engine/libopentelemetry.so /etc/centreon-engine/otl_server.json` directive. Save the form.
```

> 2. Export the poller's configuration ___with a restart___ of **Centreon Engine**.

Reviewer: ```suggestion
2. Export the poller's configuration, selecting the **Restart** option.
```
